crypto/x509v3/v3_ncons.c

```diff
@@ -64,6 +63,45 @@ ASN1_SEQUENCE(NAME_CONSTRAINTS) = {
 IMPLEMENT_ASN1_ALLOC_FUNCTIONS(GENERAL_SUBTREE)
 IMPLEMENT_ASN1_ALLOC_FUNCTIONS(NAME_CONSTRAINTS)
 
+/*
+ * We cannot use strncasecmp here because that applies locale specific rules.
+ * For example in Turkish 'I' is not the uppercase character for 'i'. We need to
+ * do a simple ASCII case comparison ignoring the locale.
+ */
```

Comment:
Please add a comment that says "that is why we use numeric constants, below" or something similar.

Reply:
And same comment in #4570 too.

Reply:
Done